content/en/ebpf.md

```diff
@@ -0,0 +1,27 @@
+---
+title: eBPF
+status: Completed
+category: architecture
+---
+
+## What it is
+
+Linux system has two spaces: kernel and user space. Kernel space is the core of the operating system and it is the only part that has unlimited access to the hardware. 
+Usually, applications reside in the user space and if they need some higher permissions, they send a request to the kernel space for that.
+For applications that require more flexibility (exp. to access hardware), there is an approach called “Linux kernel modules”. It allows extending the kernel base without adding directly to the kernel source code. 
```

Comment:
```suggestion
For applications that require more flexibility, e.g., access hardware, the kernel can be extended with eBPF via a so-called "Linux kernel modules" approach.   
```

Reply:
Are there other ways of taking a "Linus kernel module" approach that isn't eBPF?